### DISCUSSION TITLE
No appenders could be found for logger (org.apache.spark.deploy.SparkSubmit$$anon$2).

### DISCUSSION THREAD

**Discussion by asif48** [dotnet/spark#501] [Q&A]
**Describe the bug**
I have seen an article with the same subject but I am not been able to resolve my problem using that solution...

I am following an article of running [Apache Spark with asp.net C#.](https://www.red-gate.com/simple-talk/dotnet/net-development/apache-spark-for-net-developers/)
 when I ran the command from cmd I got error as described in the subject.

**Expected behavior**
System must produce the simple output as described in the blog.

**Screenshots**
![spark submit](https://user-images.githubusercontent.com/7997380/80640184-7b740400-8a7c-11ea-8315-544bc367716c.png)

**Desktop (please complete the following information):**
 - OS:  Windows 10 64 bit
 - Browser Chrome
 - Version 81.0.4044.129 (Official Build) (64-bit)

**Additional context**
I am using Apache Spark 2.4.5 with the environment variable as below spark_home=C:\spark_home\spark-2.4.5-bin-hadoop2.7
dotnet_worker_dir=C:\spark_home\Microsoft.Spark.Worker-0.10.0
hadoop_home=C:\WinUtils

rest of the path in spark_submit command is valid and accessible...I ran the spark_submit from the bin/debug/netcoreapp3.1 folder as described in the blog.

**Comment by asif48** [comment]
I have changed the line from `--class org.apache.spark.deploy.DotnetRunner` to  `--class org.apache.spark.deploy.dotnet.DotnetRunner` then it throws another error as below ...

Could not resolve CoreCLR path. For more details, enable tracing by setting COREHOST_TRACE environment variable to 1

![spark submit](https://user-images.githubusercontent.com/7997380/80646857-efb3a500-8a86-11ea-949e-97f848a06dda.png)

**Comment by imback82** [comment]
@asif48 Did you try to set `COREHOST_TRACE=1` to see what's going on?
